docs/_dg/UseCases.md

```diff
@@ -188,9 +188,9 @@ Extensions:
 
 #### Use Case 9: Increment/Decrement Quantity of Item
 
-MSS:
+[[ MSS ]]:
```

Comment:
Like OS, this is an invalid key as the glossary entry name is "MSS (Main Success Scenario)". However, I'm going to change this to "Main Success Scenario (MSS)" in another PR, so let's fix it the right way.

```suggestion
[[ main-success-scenario-mss:MSS ]]:
```